en/config_vtol/vtol_tailsitter_tuning.md

```diff
@@ -0,0 +1,66 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration documentation for "Duo-Tailsitter". This is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+
+It has two modes of operation like the other VTOLs, MC (multicopter) mode and FW (fixed wing) mode. The way it differs from the other frames is regarding how it actuates itself in those modes. In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the ailevons. In FW mode, yaw depends on differential thrust, while roll and pitch depend on the ailevons.
+
+For airframe specific documentation and build instructions see [VTOL Framebuilds](../frames_vtol/README.md).
+
+## Firmware & Basic Settings
+
+1. Run *QGroundControl*
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe (Generic VTOL Tailsitter, or Caipiroshka VTOL).
+
+
+### Flight / Transition Mode Switch
+
+You should assign a switch on your RC controller for switching between the multicopter- and fixed wing modes.
+
+:::note
+While PX4 allows flight without an RC controller, you must have one when tuning/configuring up a new airframe.
+:::
+
+This is done in [Flight Mode](../config/flight_mode.md) configuration, where you [assign flight modes and other functions](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to switches on your RC controller.
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+The MC tuning is tricky, as certain axes depend on the control surfaces, and certain ones depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+To tune the tailsitter in MC mode, we start off by setting some gains to 0. Just as a sanity check. 
+- Roll rate control ([MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = 0, [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = 0, [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = 0, [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = 1)
+- Pitch rate control ([MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = 0, [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = 0, [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = 0, [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = 1)
```

Comment:
Actually MC_PITCHRATE_X and MC_YAWRATE_X params shouldn't have an effect on the tuning of a duo tailsitter. In hover mode it uses the MC controller for the axes actuated by motors (only roll), while it uses the fixed-wing controller for pitch and yaw. So you normally tune:

For hover yaw:
FW_RR_P
FW_RR_I
FW_RR_FF (I recommend to keep this low, or even at 0)
and for hover pitch:
FW_PR_P
FW_P_I
FW_PR_FF

The convention is that control surfaces get controlled through the FW controller, and that for the FW controller the body axes are named as they are oriented in fixed-wing flight. 
I acknowledge this is insanely confusing, and thus appreciate it even more that you started with this tuning guide!